src/logic/mole-game-logic.js

```diff
@@ -29,6 +29,12 @@ export default class MoleGameLogic {
     this._enableCurrentTarget();
   }
 
+  end() {
+    let lights = this.store.data.get('lights');
+    lights.deactivateAll();
+    _.each(lights.stripIds, (id) => this._disablePanel(id));
```

Comment:
This code is now broken because of the additional stripIds Marius is adding. We can't assume it's okay to turn everything off and need to be more explicit than that.